CHANGELOG.md

```diff
@@ -6,20 +6,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### New feature support
 #### Cisco Resources
-- `cisco_aaa_authentication_login` type and provider.
-- `cisco_aaa_authorization_login_cfg_svc` type and provider.
-- `cisco_aaa_authorization_login_exec_svc` type and provider.
-- `cisco_aaa_group_tacacs` type and provider.
-- `cisco_acl` type and provider
-- `cisco_evpn_vni` type and provider.
-- `cisco_vni` type and provider.
```

Comment:
I don't think you really want to remove all of these lines? 

Is your fork too old? I'm guessing that an update of your private repo would resolve this.